csirtg_smrt/parser/zcsv.py

```diff
@@ -8,5 +8,6 @@ def __init__(self, *args, **kwargs):
         super(Csv, self).__init__(*args, **kwargs)
 
         self.pattern = re.compile(",")
+        self.add_orig= True
```

Comment:
@wesyoung FWIW, I'd be inclined (based on discovering this and our experiences with this being a surprise problem) to agree with the comment [here](https://github.com/csirtgadgets/csirtg-smrt-v1/pull/265#issuecomment-651466201) that it makes a lot of sense to remove this line and let it default to False for all parsers.  I think letting folks enable this only when needing to do some specific troubleshooting on a per-parser basis makes the most sense, this was generating lots of errors on our cluster and almost made it past us prior to moving into production.

Reply:
Went ahead and created PR #292 to address this.